### PR TITLE
Add ubuntu 20, debian 9, 10 and disable centos 8

### PR DESCRIPTION
**Pull request by ktsamis**
## What does this PR change?

Add debian 9 and 10 and disable centos 8

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- Cucumber tests were added

- [x] **DONE**

## Links

Fixes partially #13659
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by Bischoff — APPROVED** [review]
LGTM. I could not find any error, but it's easy to miss one.

Good catch for the ubuntu arch :-) .

File: testsuite/features/build_validation/init_clients/debian9_ssh_minion.feature [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    And I select "1-debian9_ssh_minion_key" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies"
+    And I check "manageWithSSH"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+    And I wait until onboarding is completed for "debian9_ssh_minion"
+
+  # WORKAROUND bsc#1181847
+  Scenario: Import the GPG keys for 9 Salt SSH minion
+    When I import the GPG keys for "debian9_ssh_minion"
```
Comment (srbarrios) [inline testsuite/features/build_validation/init_clients/debian9_ssh_minion.feature:30]: same

**Review by srbarrios — APPROVED** [review]
LGTM

File: testsuite/features/build_validation/init_clients/debian10_ssh_minion.feature [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    And I select "1-debian10_ssh_minion_key" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies"
+    And I check "manageWithSSH"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+    And I wait until onboarding is completed for "debian10_ssh_minion"
+
+  # WORKAROUND bsc#1181847
+  Scenario: Import the GPG keys for 10 Salt SSH minion
+    When I import the GPG keys for "debian10_ssh_minion"
```
[the remark this answers is not in the document]
Comment (ktsamis) [reply testsuite/features/build_validation/init_clients/debian10_ssh_minion.feature:30]: I see that this exists on all ssh minions so I think it's still necessary. I think we only removed it on salt minions.

**Pull request merged**